nfs/exports.go

```diff
@@ -176,37 +177,30 @@ func DeleteExport(directory string) (int64, error) {
 	}
 
 	if err := scanner.Err(); err != nil {
-		err = file1.Close()
-		if err != nil {
-			log.Infof("failed to close %s: %v", exportsDir, err)
-		}
 		return generation, fmt.Errorf("error reading %s: %v", exportsDir, err)
 	}
-	err = file1.Close()
+
+	err = file.Close()
```

Comment:
Do we need this file.close (or the others, including the defer below), wont "defer file.Close() on line 171" be invoked anyways when the func returns?

Reply:
ah ok, I see that now. That makes me wonder, why close the file to re-open again? I know its parsing the lines to find exports matching something, but would we lose other exports that dont match the pattern. Approving the PR, just a question however.

Reply:
Will take a look outside of this PR 👍🏽